templates/web.ipv6.template.yml

```diff
@@ -1,8 +0,0 @@
-run:
```

Comment:
I think we still need to keep this template around but make it a noop? Maybe echo a deprecation message to state that this will be removed by a given date?